### DISCUSSION TITLE
FFmpeg exited with code: 1 and signal: null (Error)

### DISCUSSION THREAD

**Discussion by JSaaS** [potmat/homebridge-google-nest-sdm#122] [Q&A]
Hi all,

I tried to get things set up yesterday. Google does not make it simple to get through their OAuth process! But, eventually, I got there. Just to be greeted with: "FFmpeg exited with code: 1 and signal: null (Error)" for all my cameras.

I am running on an RPi 4 2Gb for now. To be sure I double-checked that everything was up to date, including FFMpeg:

ffmpeg is already the newest version (8:4.3.6-0+deb11u1+rpt5)

Intermittently I get strange WebRtc-errors also: "Could not execute device command:" and further down in that message: "code":404,"message":"WebRtc session not found."
This does not seem to happen every time I try to open a camera stream, but sometimes.

So, what do you think I should check next?

Thanks in advance!

/Joakim

**Comment by joeflem** [comment]
Also getting this issue on Pi 4GB. FFMpeg is up to date, tried reinstalling plugin. Sensor events seem to be triggering but the stream for either of my cameras is throwing the error `FFmpeg exited with code: 1 and signal: null (Error)`. Using the default libx264 encoder.

One thing I did notice is that previously I had Homebridge running on my Mac and that was working fine with the exact same plugin config, but streaming when running on the Pi is broken.

**Comment by JSaaS** [comment]
Nice to know that I am not alone in this. I think it is something with the FFMpeg version somehow, but I am not very familiar with how it works. That always does things a bit harder to investigate, but I think I will learn something from it!

**Comment by potmat** [comment]
The official Homebridge pi image works fine for me: https://homebridge.io/raspberry-pi-image
